### PR TITLE
TILA-1966: Update order client

### PR DESCRIPTION
**Pull request by saaste**
I also did some refactoring:
- renamed namespace settings because it is global and not merchant API specific
- changed order client tests to be classes
- fixed the error message in product client (this was in the ticket as well)

I noticed that Order Experience API spec is not up-to-date so it could be that I need to update Order models but that'll be a task for the future. I already reported these finding to Verkkokauppa team.

**⚠️ REQUIRES CONFIGURATION CHANGES IN DEPLOYMENT ⚠️**
- [x] PR done ([PR](https://dev.azure.com/City-of-Helsinki/tilavarauspalvelu/_git/tilavarauspalvelu-pipelines/pullrequest/2113))
- [x] Library updated
- [x] Merged

### CODE REVIEW

**Review by nicobav — APPROVED** [review]
Pure awesomeness :sunglasses:

**Comment by saaste** [comment]
I'll merge this after pipeline PR is merged 🦺

**Pull request merged**
